metecho/api/migrations/0065_scratchorg_is_created.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.4 on 2020-03-31 21:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0064_user_agreed_to_tos_at"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="scratchorg",
+            name="is_created",
+            field=models.BooleanField(default=False),
+        ),
```

Comment:
@wlonk Actually, now that we have real users with real data, it's pretty important that this defaults to `True` on the initial migration. Can you make that change?

Reply:
Done.